src/lib.rs

```diff
@@ -191,6 +191,26 @@ impl Event {
         }
     }
 
+    /// Tell whether any listeners are currently notified.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use event_listener::Event;
+    ///
+    /// let event = Event::new();
+    /// let listener = event.listen();
+    /// assert!(!event.is_notified());
+    ///
+    /// event.notify(1);
+    /// assert!(event.is_notified());
+    /// ```
+    #[inline]
+    pub fn is_notified(&self) -> bool {
+        self.try_inner()
+            .map_or(false, |inner| inner.notified.load(Ordering::Relaxed) > 0)
```

Comment:
is `Relaxed` really the correct ordering for this?

Reply:
`Acquire` might be better, given that a `Release` ordering is used to store the `notified` atomic.

Reply:
yep